src/Illuminate/Mail/Mailer.php

```diff
@@ -112,6 +112,18 @@ public function alwaysFrom($address, $name = null)
 	}
 
 	/**
+	 * Set the global to address and name.
+	 *
+	 * @param  string  $address
+	 * @param  string  $name
```

Comment:
Well, was already wrong initially on the alwaysFrom() function then as I got it from there. Will do that on both.